File: app/src/main/java/com/study/trajectory_semifinal/activities/MainActivity.java
package com.study.trajectory_semifinal.activities;

import androidx.appcompat.app.AppCompatActivity;
import com.study.trajectory_semifinal.databinding.ActivityMainBinding;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/study/trajectory_semifinal/activities/MainActivity.kt
package com.study.trajectory_semifinal.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.study.trajectory_semifinal.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityMainBinding.inflate(
            layoutInflater
        )
        setContentView(binding.root)
    }
}